docs/index.rst

```diff
@@ -6,8 +6,220 @@
 The Zoo
 =======
 
+A smart service catalog providing an overview of your services' development and operations.
+
+Adding custom checks
+====================
+
+If you want to add your own checks to the zoo, you need to create your own project with files:
+
+- metadata (holds the list of errors code can trigger)
+- checks (holds the :code:`check_` functions)
+
+Both of them need to be *accessible to the zoo*.
+
+Metadata
+--------
+
+Metadata is a yaml file with a list of errors your checks yield. There are two compulsory parameters in the header:
+
+- :code:`namespace`
+- :code:`category`
+
+Then you list the errors with parameters:
+
+- :code:`id` - a unique identifier (*required*)
+- :code:`title` - a human-readable title (*required*)
+- :code:`description` - additional information about the error (*optional*)
```

Comment:
Mention that they can also pass information to the description. There is the details object that can be added when yielding the error and in the description it can be rendered.

For example

```yaml
id: something
title: something
description: |
  Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.

  {details}
```